certbot/main.py

```diff
@@ -649,6 +649,7 @@ def main(cli_args=sys.argv[1:]):
     args = cli.prepare_and_parse_args(plugins, cli_args)
     config = configuration.NamespaceConfig(args)
     zope.component.provideUtility(config)
+    cli.possible_deprecation_warning(config)
```

Comment:
This should be moved after the call to `setup_logging`. Otherwise, the only output is `No handlers could be found for logger "certbot.cli"`.